test/util/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# ******************************************************************************
+# Copyright 2017-2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+set (SRC
+    autodiff/backprop_function.cpp
+    test_tools.cpp
+    benchmark.cpp
+)
+
+include_directories(
+    ${NGRAPH_INCLUDE_PATH}
+    ${CMAKE_CURRENT_SOURCE_DIR}/..
+)
+
+add_library(ngraph_test_util SHARED ${SRC})
+install(TARGETS ngraph_test_util DESTINATION ${NGRAPH_INSTALL_LIB})  # libngraph_test_util.so
```

Comment:
The comment is incorrect, or at least correct only for certain platforms. Remove the comment.